finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+|ADVANCE |The cash advance given by Zettle to a merchant. A cash advance is a type of financing that is offered to merchants based on their sales history. The advance is paid back with monthly down payments.
+|ADVANCE_DOWNPAYMENT |A down payment on a previously paid out cash advance.
+|ADVANCE_FEE_DOWNPAYMENT |The netting of a cash advance fee.
+|BANK_ACCOUNT_VERIFICATION (Deprecated) |N/A
+|CARD_PAYMENT |A card payment. Contains a reference to the card payment in the Purchase API.
+|CARD_PAYMENT_FEE |The commission part of a card payment.
+|CARD_PAYMENT_FEE_REFUND |The commission part of a refund.
+|CARD_REFUND |A card refund. <!-- Is it not obvious? -- Accompanied by CARD_PAYMENT_FEE_REFUND. --> Contains a reference to the card payment refund in the Purchase API.
+|CASHBACK |Money given to a merchant to retroactively adjust the card payment fee rate.
+|CASHBACK_PAYOUT (Deprecated) |N/A
+|EMONEY_TRANSFER (Deprecated) |N/A
+|FAILED_PAYOUT |A previous payout transaction has failed and been made void. The payout money is returned to the merchant's Zettle liquid account.
+|FEE_DISCOUNT_REVOCATION |An internal reclaim of an outstanding fee discount that is not consumed within a certain time frame. As these funds are reclaimed from a special fee discount account, the transaction will not be visible in the Zettle liquid account of the merchant.
+|FROZEN_FUNDS |The money that is frozen to cover a chargeback. When the issuing back initiates a chargeback, the money will be removed from the merchant's liquid account and marked as frozen to cover the chargeback. If the chargeback is later revoked, the money will be returned to the merchants liquid account with a new and positive transaction of the same type. It effectively makes the initial FROZEN_FUNDS transaction void.
+|INVOICE_PAYMENT |An invoice payment.
+|INVOICE_PAYMENT_FEE |An invoice payment fee.
+|PAYMENT |An alternative third-party payment method where Zettle handles the funds. For example, PayPal QR code and Klarna QR code.
+|PAYMENT_FEE |The fee for a third-party payment method. For example,PayPal QR code and Klarna QR code.
+|PAYOUT |A payout from the merchant's Zettle liquid account to the merchant’s bank account .  
+|TELL_FRIEND (Deprecated) |N/A
+|VOUCHER_ACTIVATION |Used when activating a voucher. The money is inserted to the merchant’s fee discount account instead of preliminary and liquid accounts.
+ 
+</details>
+
+
+### Responses
+<details open="true">
+<summary>HTTP status codes</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the account transactions are successfully fetched.  
+|403 Forbidden |Returned when you are not authorised to fetch the account transactions.
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request.  
+</details>
+
+<details open="true">
+<summary>Response attributes</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the account transactions. 
```

Comment:
It's not an object it's an array of objects.
"Information about the account transactions" doesn't make sense since there is no additional information about transactions.

Suggested:
"Array of transactions for given account